packages/app/src/overview/components/overview-screens/pure-overview/pure-overview.component.tsx

```diff
@@ -243,6 +249,13 @@ export const PureOverview: FunctionComponent<Props> = ({
     }
   }, [restoreDeviceState])
 
+  const history = useHistory()
+
+  const onRetry = () => {
+    // TODO: do some logic to retry connection
+    history.push(URL_ONBOARDING.connecting)
```

Comment:
🟡 
is it TODO or not? 🤔 

One of your AC is _"The failure modal allows triggering sync again"_

Reply:
Fixed in [6e9f68e](https://github.com/mudita/mudita-center/pull/642/commits/6e9f68e9693fe7bc9971fdb61d6aa68c65fe95c2)